src/main.ts

```diff
@@ -28,13 +51,11 @@ async function run() {
       core.warning("⚠️ Not a pull request, skipping PR body checks");
     } else {
       if (bodyContains || bodyDoesNotContain) {
+        const PRBody = context?.payload?.pull_request?.body;
```

Comment:
Again, the action can't continue if context does not exist; if payload does not exist, either. It's probably best to check early, and then just go with it, don't you think so?